HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -64,6 +64,8 @@
 #define HDMICECSINK_METHOD_SEND_GIVE_AUDIO_STATUS          "sendGetAudioStatusMessage"
 #define HDMICECSINK_METHOD_GET_AUDIO_DEVICE_CONNECTED_STATUS   "getAudioDeviceConnectedStatus"
 #define HDMICECSINK_METHOD_REQUEST_AUDIO_DEVICE_POWER_STATUS   "requestAudioDevicePowerStatus"
+#define HDMICECSINK_METHOD_REQUEST_CURRENT_LATENCY	"requestCurrentLatency"
```

Comment:
Pls update semantic versioning minor version and changelog since this has new API changes. Also update minor version macro.